lib/main.coffee

```diff
@@ -14,7 +14,9 @@ isMarkdownPreviewView = (object) ->
 atom.deserializers.add
   name: 'MarkdownPreviewView'
   deserialize: (state) ->
-    createMarkdownPreviewView(state) if state.constructor is Object
+    if state.constructor is Object
```

Comment:
I don't think this check is needed anymore, it was probably old and from when deserialization behaved differently.